pkg/health/types.go

```diff
@@ -26,16 +26,39 @@ import (
 	"time"
 )
 
+// StorageIndicatorName is the name of storage indicator
+const StorageIndicatorName = "storage"
+
+// PlatformIndicatorSuffix is the suffix for a platform type indicator
+const PlatformIndicatorSuffix = "_platforms"
+
+// indicatorNames is a list of names of indicators which will be registered with default settings
+// as part of default health settings, this will allow binding them as part of environment.
+// If an indicator is registered but not specified in this list, it will be configured with
+// default settings again, but this defaults could be overridden only via application.yml,
+// env variables and pflags won't have any effect. If an indicator is specified in this list
+// but later not registered nothing will happen.
+var indicatorNames = [...]string{
```

Comment:
what about indicators outside Peripli? who will add them here?

Reply:
They will maintain separate list of indicator names and extend default settings with them while configuring their settings. Other approach is to export this array lets discuss which is better.